public/locales/gu-IN.json

```diff
@@ -0,0 +1,18 @@
+{
+  "we": "અમે",
+  "making": "રસીકરણ કેન્દ્રો શોધવા માટેનું તમારું કામ",
+  "locate": "સરળ કરી રહ્યા છીએ",
+  "description": "અમે દેશભરમાં આવેલા રસીકરણ કેન્દ્રોની એક સૂચી બનાવી રહ્યા છીએ જેથી તમે તમારી માટે કે તમારા પ્રિયજનો માટે રસી મેળવવા ઝડપથી અને સરળતાથી રસીકરણ કેન્દ્ર વિશેની માહિતી મેળવી શકો.",
+  "join": "ટીમમાં જોડાઓ",
+  "learn": "વધુ જાણો",
+  "how": "અમે આ કેવી રીતે કરી રહ્યા છીએ?",
+  "with": "18 વર્ષથી વધુ ઉંમરની વ્યક્તિઓ માટે કોવિડ-૧૯ રસીકરણ આપવાનું શરૂ થતાની સાથે જ દેશભરમાં જાહેર આરોગ્ય કેન્દ્રો પર આમ જનતાનો ધસારો થઈ જશે. અમે તમને ઓછી ભીડવાળા રસીકરણ કેન્દ્રો પર રસીની ઉપલબ્ધતા વિશે જાણકારી મેળવવા માટે સહાય કરવા માંગીએ છીએ.",
+  "our": "આ સૂચીમાં નવીનતમ માહિતીઓ મળી રહે તે માટે અમારા ૧૦૦થી વધુ સ્વયંસેવકો કાર્યરત છે. જો તમે અમારા આ પ્રયત્નોમાં જોડાશો તો અમને પણ આનંદ થશે કારણ કે તેનાથી સામાન્ય નાગરિકોને દરેક રસીકરણ કેન્દ્રો પર ફોન કરવાની જરૂર નહીં પડે, તેઓ આપણી આ સૂચિની મદદથી નજીકના રસીકરણ કેન્દ્રો વિશે માહિતી મેળવી શકશે.",
+  "help": "તમે કઈ રીતે મદદ કરી શકો છો?",
+  "glad": "તમે પૂછ્યું તે બદલ આનંદ થયો! અમે સંપૂર્ણપણે સમાજ અને સ્વયંસેવકોની મદદથી કાર્ય કરીએ છીએ. અમારી પાસે એવા અમુક વિસ્તારો છે જેને માટે અમે સ્વયંસેવકો શોધી રહ્યા છીએ. તમે અમને આ રીતે મદદ કરી શકો છો: -",
+  "tech": "ટેક્નોલોજી:",
+  "techteam": " જો તમારી પાસે દરરોજ 2-3 કલાકનો સમય અને React JS અથવા Mongo જેવી કુશળતા હોય, તો તમે અમારી તકનીકી ટીમ સાથે જોડાઈ શકો છો.",
+  "calling": "કોલિંગ સેન્ટરો:",
+  "callteam": "જો તમારી પાસે દરરોજ ૨૦ મિનિટનો સમય હોય, તો તમે દરરોજ 5-6 રસીકરણ કેન્દ્રો પર ફોન કરવા માટે સાઇન અપ કરી શકો છો. અને સંખ્યાબંધ લોકોને વિવિધ માહિતીઓ પૂરી પાડવા માટે સહાય કરી શકો છો. જો તમને લાગતું હોય કે અમે હજુ પણ કંઈક સારું કરી શકીએ છીએ તો તમે અમને Instagram કે Twitter પર મેસેજ કરી શકો છો. અને તમારો આ પ્રતિસાદ અમને મદદરૂપ પણ થશે.",
+  "disclaimer": "અમે સ્વયંસેવકોની મદદથી આ સંસ્થા ચલાવીએ છીએ અને સચોટ માહિતીઓ આપવાનો પ્રયાસ કરીએ છીએ પરંતુ માહિતી અંગેની ચોકસાઈ માટે કોઈ કાનૂની બાંયધરી આપતા નથી. અમે તબીબી સલાહ પ્રદાન કરતા નથી."
+}
```

Comment:
can we maintain single file for gujrati language.

Reply:
Hey @kshitizshakya @chavda-bhavik can ignore this comment for now, 

The thing is making the `gu-IN` file is not required, but since you have it in now, it's not a problem, we'll clean it up in a future release